modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -1,38 +1,127 @@
-= Controlling File Versions and Aging
+= Controlling File Versions
 
-The Versions app (files_versions) expires old file versions
-automatically to ensure that users don’t exceed their storage quotas.
-This is the default pattern used to delete old versions:
+== How Versions are created
 
-* For the first second we keep one version
-* For the first 10 seconds ownCloud keeps one version every 2 seconds
-* For the first minute ownCloud keeps one version every 10 seconds
-* For the first hour ownCloud keeps one version every minute
-* For the first 24 hours ownCloud keeps one version every hour
-* For the first 30 days ownCloud keeps one version every day
-* After the first 30 days ownCloud keeps one version every week
+Every time when a file is written to the storage the versions app
+(files_versions) creates a new backup copy of the file. These backups are
+stored inside a folder `files_versions` inside the users root folder.
+The app will add the suffix `.v` followed by the unix timestamp of the creation
+date of the backup copy.
+
+....
+.
+├── files
+│   └── welcome.txt
+└── files_versions
+    ├── welcome.txt.v1556203470
+    ├── welcome.txt.v1556203501
+    └── welcome.txt.v1556203567
+....
+
+Versions are displayed in the WebUI in the details view in the right sidebar if
+you click on the file row in the file listing. You can restore the current file
+to one of the earlier backup copies in the list by clicking on the restore icon
+of the specific version.
+
+image::files-versions.png[File Versions in the WebUI]
+
+== How Versions are expired
+
+The versions app expires old file versions
+automatically to ensure that users do not exceed their storage quotas.
+This is done by automatic background jobs which clean up the versions following
+a specific pattern.
+
+==== This is the default pattern used to delete old versions:
+
```

Comment:
I think it would be even better to read if we wrote "For the last second since the background job was executed"

Reply:
or something similar